frontend/src/components/footer.jsx

```diff
@@ -11,22 +11,29 @@ export default class footer extends Component {
             <footer className='footer py-3'>
                 <div className='container'>
                     <nav>
-                        <ul className='row'>
+                        <ul className='row menu-text'>
                             <li className='col footer_link'>
-                                <Link to="/about">About Us</Link>
+                                <Link to="/about" style={{ color: 'white', fontSize: "12px" }} className="foot-text">About</Link>
```

Comment:
This styling should be in a class; it's applying to a number of elements, and generally we always want to avoid in-line styling.